doc/en-us/dev/js/event_listener.md

```diff
@@ -1,54 +1,54 @@
-# 事件监听  
+# Event Listening
 
-事件是PNX中插件执行工作的主要方式之一，本章节将通过带您制作一个简单的屏蔽词插件带您学会使用PNX-JS进行事件监听。  
+Events are one of the main ways plugins perform their work in PNX, and this section will take you through the process of learning to use PNX-JS for event listening by taking you through the creation of a simple blockword plugin.
 
-## 什么是事件  
+## What is an event  
 
-每当服务器中发生了特定的行为，一个事件对象就会产生。事件是对某种行为或动作的描述，比如玩家登录、
-玩家离开、玩家发送消息等等。种种的行为或动作都对应一个事件对象作为它们的抽象，事件对象中带有这个行为或者动作中的一些
-关键信息，比如玩家发送消息事件对象中就带有发送事件的玩家对象、发送的消息字符串等等，而玩家对象又包含这个玩家的
-位置、生命值、物品栏等等数据。  
+An event object is generated whenever a specific action occurs in the server. An event is a description of some behavior or action, such as a player logging in, a
+player leaves, player sends a message, and so on. Each kind of behavior or action has an event object as its abstraction, and the event object carries some key information about the behavior or action.
+The event object contains some key information about the behavior or action, e.g., the player object that sent the event, the message string sent, etc., and the player object contains the player's
+The player object contains the player's location, life value, item bar, etc.
```

Comment:
Here seems to be a repetition of the content here

Reply:
fix next commit

Reply:
11790750ed1420b783524860fab09c4dcbf3704d